src/modules/75-cf/components/FlagActivation/LinkJiraIssueModal.tsx

```diff
@@ -0,0 +1,190 @@
+      } catch (e) {
+        showError(getErrorMessage(e), undefined, 'cf.featureFlags.jira.errorMessage')
+      } finally {
+        patch.feature.reset()
+      }
+    })
+  }
+
+  return (
+    <>
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }}>
+        <RbacButton
+          data-testid="add-jira-issue-modal"
+          minimal
+          icon="small-plus"
+          size={ButtonSize.SMALL}
+          font={{ size: 'small' }}
+          style={{ padding: 0 }}
+          text={getString('cf.featureFlags.jira.newJiraIssueButton')}
+          onClick={() => openModal()}
+          permission={{
+            resource: { resourceType: ResourceType.FEATUREFLAG },
+            permission: PermissionIdentifier.EDIT_FF_FEATUREFLAG
+          }}
+          featuresProps={{
+            featuresRequest: {
+              featureNames: [FeatureIdentifier.MAUS]
+            }
+          }}
+        />
+      </Layout.Horizontal>
+
+      <Formik
+        initialValues={{
+          selectedJiraIssueKey: ''
+        }}
+        formName="addJiraIssueDialog"
+        enableReinitialize={true}
+        validateOnChange
+        validateOnBlur
+        validationSchema={yup.object().shape({
+          selectedJiraIssueKey: yup.string().trim().required(getString('cf.featureFlags.jira.jiraIssueRequiredError'))
+        })}
+        onSubmit={formData => {
+          handleSubmit(formData.selectedJiraIssueKey)
+        }}
+      >
+        {formikProps => (
+          <FormikForm>
+            <ModalDialog
+              isOpen={isModalDisplayed}
+              enforceFocus={false}
+              onClose={hideModal}
+              title={getString('cf.featureFlags.jira.jiraModalTitle')}
+              height={290}
+              footer={
+                <Layout.Horizontal spacing="small">
+                  <Button
+                    variation={ButtonVariation.PRIMARY}
+                    disabled={patchLoading || jiraIssuesLoading}
+                    text={getString('add')}
+                    intent="primary"
+                    onClick={() => formikProps.submitForm()}
+                  />
+                  <Button
+                    variation={ButtonVariation.TERTIARY}
+                    disabled={patchLoading || jiraIssuesLoading}
+                    text={getString('cancel')}
+                    onClick={() => {
+                      hideModal()
+                      formikProps.resetForm()
+                    }}
+                  />
+                </Layout.Horizontal>
+              }
+            >
+              <FormInput.Select
+                label={getString('cf.featureFlags.jira.inputLabel')}
+                name="selectedJiraIssueKey"
+                items={jiraItems}
+                placeholder={getString('cf.featureFlags.jira.inputPlaceholder')}
+                selectProps={{
+                  loadingItems: jiraIssuesLoading
+                }}
```

Comment:
You're probably seeing the popover for the list of result tucking inside the body of the modal. You can fix that by telling the select to use a portal to display
```suggestion
                selectProps={{
                  loadingItems: jiraIssuesLoading,
                  usePortal: true
                }}
```